template/manifests/base/deployment.yaml

```diff
@@ -1,8 +1,12 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
+  annotations:
+    zenoss.spinnaker_deploy: "true"
+    # zenoss.zing.deploy.istio: "true" // Uncomment this if you're exposing a public API.
   labels:
     app: {{Name}}
+    project: {{Name}}
```

Comment:
The `project` label should be added to the pod template (spec/template/metadata/labels) too. That way the pods created by the deployment will also have the label.